app/src/main/kotlin/com/wire/android/ui/WireActivity.kt

```diff
@@ -182,7 +182,7 @@ class WireActivity : AppCompatActivity() {
         }
     }
 
-    override fun onNewIntent(intent: Intent?) {
+    override fun onNewIntent(intent: Intent) {
```

Comment:
In newest version, `ComponentActivity` got refactored to kotlin so intent is not nullable anymore